### PR TITLE
tt: change working directory

### LINKED ISSUE #570: Change `work dir`
The `work dir` should be moved to directory with the application sources. 
This change should include the following effects:
* by default, artifacts from the application will be created in the source directory
* a directory tree (`var/log`, `var/run`, `var/lib`) will be created in the source directory (now it's created in the root of the env)
* in `tt,yaml` will separate the common section and the application section. In the first case, the relative path will refer to the env root, and in the second case it will refer to the application source directory

### PR DESCRIPTION
**Pull request by psergee**
- Change work dir to application source location. In case of
single instance application, separate directory will be created
in instances enabled directory.
- Re-work application files layout and naming. `app` section 
relative paths are considered relative to working directory, 
which is an application source directory. Application name
sub-directory no longer used for relative paths.
- Support new layout in pack.

Closes #570

### CODE REVIEW

File: cli/pack/common.go [hunk 1]
```diff
@@ -63,6 +51,63 @@ var (
...
+	}
+
+	if strings.HasPrefix(src, ".git") ||
+		strings.Contains(src, "/.git") {
+		return true, nil
+	}
+	return false, nil
+}
+
+// prepeareArtifactFilters prepare a slice of filter for runtime artifacts.
```
Comment (better0fdead) [inline cli/pack/common.go:72]:
```suggestion
// prepeareArtifactFilters prepares a slice of filter for runtime artifacts.
```

**Review by better0fdead — CHANGES_REQUESTED** [review]
Hi, thx for patch. Please add a Changelog entry and description to the second commit,

**Review by LeonidVas — APPROVED** [review]
Hi! Thank you for the patchset.
LGTM!

File: cli/pack/common.go (hunk 1 above)
Comment (psergee) [reply cli/pack/common.go]: Fixed

**Comment by psergee** [comment]
> Hi, thx for patch. Please add a Changelog entry and description to the second commit,

The change log notes are about the changes, not commits. `pack` command changes are a part of layout and naming update. Since there are a lot of changes for the `pack`, they are in a separate commit.

**Review by better0fdead — APPROVED** [review]

**Pull request merged**